dev/new-rule/templates/doc.md

```diff
@@ -0,0 +1,53 @@
+## (rule name goes here)
```

Comment:
Can you update the test suite that confirms new rules are created with all the right "things", to also check for these placeholders (and if found fail the test).

Reply:
I'd like to split this into a different issue (more for my own sprint purposes) if that's okay.

Reply:
https://github.com/ember-template-lint/ember-template-lint/issues/1119